### ISSUE TITLE
Deprecated Multisite and PHPCS Documentation

### ISSUE THREAD

**Issue by jhoscar1** [acquia/blt#1984]
My system information:
* Operating system type: [MacOS]
* Operating system version: [Sierra]
* BLT version: [8.9.2]

Output of `blt doctor`:
```
Drupal VM was detected. Running blt doctor inside of VM...
Executing command cd /var/www/comcast-ch/docroot/.. && /var/www/comcast-ch/docroot/../vendor/bin/drush cc drush && /var/www/comcast-ch/docroot/../vendor/bin/drush --include=/var/www/comcast-ch/docroot/../vendor/acquia/blt/drush blt-doctor -r /var/www/comcast-ch/docroot/../docroot inside of Drupal VM...
'drush' cache was cleared.                                                                                              [success]
+----------------------+----------------------------------------------------------------------------------+
| Property             | Value                                                                            |
+----------------------+----------------------------------------------------------------------------------+
| %paths               |                                                                                  |
|  - %root             | /var/www/comcast-ch/docroot                                                      |
|  - %site             | sites/default                                                                    |
|  - %modules          | modules                                                                          |
|  - %themes           | themes                                                                           |
|  - %config-sync      | /var/www/comcast-ch/config/default                                               |
|  - %files            | sites/default/files                                                              |
|  - %temp             | /tmp                                                                             |
| admin-theme          | cch_editorial                                                                    |
| blt-version          | 8.9.2                                                                            |
| bootstrap            | Successful                                                                       |
| config-sync          | /var/www/comcast-ch/config/default                                               |
| db-driver            | mysql                                                                            |
| db-hostname          | localhost                                                                        |
| db-name              | drupal                                                                           |
| db-password          | drupal                                                                           |
| db-port              | 3306                                                                             |
| db-status            | Connected                                                                        |
| db-username          | drupal                                                                           |
| dev-desktop-enabled  | false                                                                            |
| drupal-settings-file | sites/default/settings.php                                                       |
| drupal-version       | 8.3.7                                                                            |
| drupal-vm-enabled    | true                                                                             |
| drush-alias-files    | /var/www/comcast-ch/docroot/../drush/site-aliases/aliases.drushrc.php            |
|                      | /var/www/comcast-ch/docroot/../drush/site-aliases/comcast-ch.aliases.drushrc.php |
|                      | /var/www/comcast-ch/docroot/../drush/site-aliases/comcastacsf.aliases.drushrc.ph |
|                      | p                                                                                |
|                      | /var/www/comcast-ch/docroot/../drush/site-aliases/example.acsf.aliases.drushrc.p |
|                      | hp                                                                               |
|                      | /var/www/comcast-ch/docroot/../drush/site-aliases/example.local.aliases.drushrc. |
|                      | php                                                                              |
| drush-conf           | /var/www/comcast-ch/drush/drushrc.php                                            |
| drush-script         | /var/www/comcast-ch/vendor/drush/drush/drush.php                                 |
| drush-temp           | /tmp                                                                             |
| drush-version        | 8.1.13                                                                           |
| files                | sites/default/files                                                              |
| install-profile      | cch_lightning                                                                    |
| local-drushrc        | /var/www/comcast-ch/docroot/sites/default/local.drushrc.php                      |
| local-settings       | /var/www/comcast-ch/docroot/sites/default/settings/local.settings.php            |
| modules              | modules                                                                          |
| php-bin              | /usr/bin/php                                                                     |
| php-conf             | /etc/php/7.0/cli/php.ini                                                         |
| php-mysql            | /var/run/mysqld/mysqld.sock                                                      |
| php-os               | Linux                                                                            |
| root                 | /var/www/comcast-ch/docroot                                                      |
| shell                |                                                                                  |
| site                 | sites/default                                                                    |
| temp                 | /tmp                                                                             |
| theme                | cch_editorial                                                                    |
| themes               | themes                                                                           |
| uri                  | http://comcast-ch.local                                                          |
| user                 |                                                                                  |
+----------------------+----------------------------------------------------------------------------------+
+--------------------------+-------------------------------------------------------------+
| Check                    | Outcome                                                     |
+--------------------------+-------------------------------------------------------------+
| checkDocrootExists       | Found docroot.                                              |
| checkCoreExists          | Drupal core exists                                          |
| checkSettingsFile        | BLT settings are included in settings file.                 |
| checkLocalSettingsFile   | Found your local settings file.                             |
| checkLocalDrushFile      | Found your local drush settings file.                       |
| checkUri                 | $options['uri'] is set.                                     |
| checkUriResponse         | Received a response from site http://comcast-ch.local.      |
| checkFileSystem:%files   | Public files directory is writable.                         |
| checkFileSystem:%private | Private files directory is not set.                         |
| checkFileSystem:%temp    | Temporary files directory is writable.                      |
| checkDbConnection        | Connected to database.                                      |
| checkDrupalBootstrapped  | Bootstrapped Drupal via drush.                              |
| checkDrupalInstalled     | Drupal is installed.                                        |
| checkCaching:page        | Drupal cache is disabled.                                   |
| checkCaching:css         | CSS preprocessing is disabled.                              |
| checkCaching:js          | JS preprocessing is disabled.                               |
| checkCiConfig            | Git remotes are set in project.yml.                         |
| checkComposer:require    | acquia/blt is in composer.json's require object.            |
| checkBehat:exists        | Behat local settings file exists.                           |
| checkBehat:root          | Behat drupal_root is set correctly for Drupal VM.           |
| checkProjectYml:keys     | project.yml has no deprecated keys.                         |
| checkAcsfConfig          | BLT settings are included in your pre-settings-php include. |
| checkDrupalVm:alias      | drush.aliases.local exists your drush aliases file.         |
| checkDrupalVm            | Drupal VM is configured correctly.                          |
| checkPhpDateTimezone     | PHP setting for date.timezone is correctly set              |
+--------------------------+-------------------------------------------------------------+
```

It seems that the documentation for the `multisite` property in https://blt.readthedocs.io/en/stable/readme/multisite/ is no longer correct as of the aforementioned blt version. According to `./vendor/acquia/blt/config/build.yml` on lines 89-94 the correct format for the multisite property is now

```
multisites:
  - default
```
as opposed to the format in the documentation written as follows:

```
multisite:
  name:
    - default
```
This change seems as though it may impact additional documentation related to Multisite Tasks as well.

Additionally, the process for "Adding / overriding filesets" on the [Extending / Overriding BLT](http://blt.readthedocs.io/en/latest/readme/extending-blt/) section, references using Finder while the current version of blt seems to have switched to using PHPCS. Thank you very much!

**Comment by grasmash** [comment]
Thanks. The reference to `multisites` has been fixed in 8.x branch, you can find the documentation here: https://blt.readthedocs.io/en/8.x/readme/multisite/

**Comment by notacouch** [comment]
We're using `8.9@stable`, not sure if this has made its way in yet? Even in 8.9.8 the multisite.md has the old documentation.

**Comment by danepowell** [comment]
@grasmash the documentation is correct on 9.1.x, but it looks like we need to backport the documentation fix to 8.9.x and 9.x, do you concur?
